tests/DevApps/AzureFunctions/AzureFunctions.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
VS is a bit interesting, so via the file explorer, this project is in a folder called "AzureFunctions" but when you open it in VS, it looks like this:
![image](https://github.com/AzureAD/microsoft-identity-web/assets/19942418/81e62edf-ec8a-4da3-ae4f-9b071495a071)

Not in a folder. So you need to right click on the DevApps folder and select "add new folder" and name it the same as the one in the file explorer `AzureFunctions`

small nit, but it will keep things more organized 😸